### ISSUE TITLE
I cant see who I am following

### ISSUE THREAD

**Issue by daleharvey** [mikeal/gatherbugs#28]
I signed up and connected with twitter, I got a notification that nuno was having dinner somewhere, but clicking it didnt do anything (#20)

When I access followers I dont see anything except a page "Didnt find any upcoming gatherings nearby! Why dont you create one", I expect to see somewhere that tells me I am actually following nuno / who I am following and possibly add / remove people

**Comment by mikeal** [comment]
hrm......

nuno must have created an event without a location.

we use your graph from twitter directly so we don't manage the follow relationships at all. we just see who you already follow on twitter/facebook and see if they are gather users and show you events that are near you from those people.

we should disable notifications if there is no place set.

**Comment by daleharvey** [comment]
Cool, I think it makes sense to leave the following / unfolowing to facebook etc, but I would still like to be given some idea of whos gathers I will receive and whos mine will be sent to

**Comment by mikeal** [comment]
yup, it's not difficult to surface this. we put stuff like this on the back burner for the initial release because we wanted to nail the initial experience of everything "just working".

it's on our todo list now :)
